src/commands/cloudmanager/magento/bin-magento/maintenance/status.js

```diff
@@ -0,0 +1,57 @@
+/*
+Copyright 2021 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+const BaseCommerceCommand = require('../../../../../base-commerce-command')
+const { getProgramId } = require('../../../../../cloudmanager-helpers')
+const { cli } = require('cli-ux')
+const commonFlags = require('../../../../../common-flags')
+
+class MaintenanceStatusCommand extends BaseCommerceCommand {
```

Comment:
Obviously not for now, but something to consider in the future is some kind of dynamic generation of these classes since I expect future commands will be very boilerplatey. https://github.com/oclif/oclif/issues/110 suggests there's a way to do this.